.github/workflows/docker-build-static.yml

```diff
@@ -488,10 +497,40 @@ jobs:
 
       - name: Setup QEMU
         uses: docker/setup-qemu-action@v3
-      - uses: docker/setup-buildx-action@v3
+      - name: Setup buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          driver-opts: network=host
+
+      - name: Docker ${{ matrix.container }} Base meta
+        id: meta-base
+        uses: docker/metadata-action@v5
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            localhost:5000/lldap/lldap
+          tags: ${{ matrix.container }}-base
+
+      - name: Build ${{ matrix.container }} Base Docker Image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          #push: ${{ github.event_name != 'pull_request' }}
```

Comment:
Uncomment

Reply:
This part is for our dummy registry, it doesn't really push to the real registry. Then enable `#push: ${{ github.event_name != 'pull_request' }}` expect to fail in PR, we already know will work fine. Going to add some comments.